Testing/SystemTests/tests/framework/MonteCarloAbsorptionCorrTest.py

```diff
@@ -0,0 +1,52 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+"""
+MonteCarloAbsorption is used within various workflow algorithms eg PaalmanPingsMonteCarloAbsorption
+but it can be used directly by end users if they first set up a sample shape using SetSample or
+LoadSampleShape. These tests cover this mode of use
+"""
+
+import systemtesting
+import mantid.simpleapi as mantid
+
+
+class CheckSampleOnlyAndEnvOnlyMatch(systemtesting.MantidSystemTest):
+
+    abs_e = None
+    abs_s = None
+
+    def runTest(self):
```

Comment:
I think there is copy-pasted code in the test, both are run as container.

Reply:
> I think there is copy-pasted code in the test, both are run as container.

Aaaargh! Have just fixed